website/docs/users-sources/groups/manage_groups.md

```diff
@@ -25,9 +25,9 @@ To create a super-user, you need to add the user to a group that has super-user
 
 ## Modify a group
 
-To edit the group's name, parent group, whether or not the group is for superusers, associated roles, and any custom attributes, click the Edit icon beside the role's name. Make the changes, and then click **Update**.
```

Comment:
I wouldn't remove the first sentence since it still applies for generic updating, the new sentence is an addition for superuser status